option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message LegionOperateRequest
{
 required U64Type playerid = 1;
}
